ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -1062,7 +1062,7 @@ objects:
           wait
 
           echo "Deprovisioning cluster ..."
-          openshift-install --dir /tmp/artifacts/installer destroy cluster
+          openshift-install --dir /tmp/artifacts/installer destroy cluster || exit 1
```

Comment:
Not sure if this makes any difference here - no other actions are happening after `destroy cluster`

It does make sense in UPI AWS template of course - I suppose this is done to have all templates in sync?

Reply:
Hah, I didn't even think of that, just blindly `grep`ed.  Will reroll...